nodejs/eks/nodegroup.ts

```diff
@@ -709,6 +709,31 @@ export function createManagedNodeGroup(name: string, args: ManagedNodeGroupOptio
         roleArn = args.nodeRoleArn;
     }
 
+    // Check that the nodegroup role has been set on the cluster to
+    // ensure that the aws-auth configmap was properly formed.
+    const roleExistsInNodeAccess = pulumi.all([
+        core.instanceRoles,
+        roleArn,
+    ]).apply(([roles, rArn]) => {
+        // Map out the ARNs of all of the instanceRoles.
+        const roleArns = roles.map(role => {
+            return role.arn.apply(a => a);
```

Comment:
It seems like you could just return `role.arn` here rather than needing the `apply(a => a)`